src/platform/darwin_openssl.c

```diff
@@ -123,7 +123,9 @@ CxPlatTlsVerifyCertificate(
         goto Exit;
     }
 
-    SSLPolicy = SecPolicyCreateSSL(true, SNIString);
+    SSLPolicy = SecPolicyCreateSSL(
+        (CredFlags & QUIC_CREDENTIAL_FLAG_CLIENT) ? false : true,
+        SNIString);
```

Comment:
nit: when the arguments are on a different line than the function call, please put the function call on its own line too.
```suggestion
    SSLPolicy =
        SecPolicyCreateSSL(
            (CredFlags & QUIC_CREDENTIAL_FLAG_CLIENT) ? FALSE : TRUE,
            SNIString);
```
Also, this is `c` code, so `TRUE` or `FALSE` please.